gamemode/modules/base/cl_fonts.lua

```diff
@@ -2,22 +2,28 @@
 The fonts that DarkRP uses
 ---------------------------------------------------------------------------]]
 local function loadFonts()
-    local tahoma = system.IsLinux() and "DejaVu Sans" or "Tahoma"
-    local tahomaSize = system.IsLinux() and fp{fn.Flip(fn.Add), 1} or fn.Id
+    local robotoSize = system.IsLinux() and fp{fn.Flip(fn.Add), 1} or fn.Id
+
+    surface.CreateFont("Target", {
```

Comment:
I like your changes, but calling a font "Target" these days is a dangerous step.
It could override fonts with the same name from other addons.